`dotnet build-server shutdown` stalls on kubernetes GitLab runners for some reason.

---
diff --git a/repo-projects/Directory.Build.targets.orig b/repo-projects/Directory.Build.targets
index 8c5073b..ba03b05 100644
--- a/repo-projects/Directory.Build.targets.orig
+++ b/repo-projects/Directory.Build.targets
@@ -781,20 +781,6 @@
           Condition="'$(IsUtilityProject)' != 'true' and
                      '$(CleanWhileBuilding)' == 'true' and
                      Exists('$(RepoArtifactsDir)')">
-    <!--
-      Some repositories (WinForms) use source generators that open files manually and keep file handles open on the compiler server (CsWin32).
-      These source generators are written incorrectly (a source generator should never do IO itself),
-      but the required features to read in binary files as input to a source generator don't exist.
-      To work around these source generators, shut down the compiler server so we can delete the obj directories.
-      Don't fail the build when shutdown returns a failure.
-
-      Don't run when source building to prevent the build from hanging indefinitely - https://github.com/dotnet/source-build/issues/4796
-    -->
-    <Exec Command="$(DotNetTool) build-server shutdown --vbcscompiler"
-          Condition="'$(DotNetBuildSourceOnly)' != 'true'"
-          EnvironmentVariables="NUGET_PACKAGES=$(RepoArtifactsPackageCache)"
-          IgnoreStandardErrorWarningFormat="true"
-          IgnoreExitCode="true" />
 
     <Message Text="DirSize After Building $(RepositoryName)" Importance="High" />
     <Exec Command="$(DiskUsageCommand)" />
diff --git a/eng/source-build-toolset-init.sh.orig b/eng/source-build-toolset-init.sh
index dd5a1eb..ced3a9a 100755
--- a/eng/source-build-toolset-init.sh.orig
+++ b/eng/source-build-toolset-init.sh
@@ -135,10 +135,8 @@ function source_only_toolset_init() {
     initSourceOnlyBinaryLog="/bl:\"$log_dir/init-source-only.binlog\""
   fi
 
-  "$_InitializeBuildTool" build-server shutdown --msbuild
   MSBuild-Core "$repo_root/eng/init-source-only.proj" $initSourceOnlyBinaryLog "${properties[@]}"
   # kill off the MSBuild server so that on future invocations we pick up our custom SDK Resolver
-  "$_InitializeBuildTool" build-server shutdown --msbuild
 
   local bootstrapArcadeDir=$(cat "$repo_root/artifacts/toolset/bootstrap-sdks.txt" | grep "microsoft.dotnet.arcade.sdk")
   local arcadeBuildStepsDir="$bootstrapArcadeDir/tools/"
diff --git a/src/roslyn/eng/build.sh.orig b/src/roslyn/eng/build.sh
index a87e4b2..a2bfe2d 100755
--- a/src/roslyn/eng/build.sh.orig
+++ b/src/roslyn/eng/build.sh
@@ -226,10 +226,6 @@ function MakeBootstrapBuild {
   echo "Cleaning Bootstrap compiler artifacts"
   dotnet clean "$project_path"
 
-  if [[ "$node_reuse" == true ]]; then
-    dotnet build-server shutdown
-  fi
-
   # return value
   _MakeBootstrapBuild=$dir
 }
